content/docs/userspace/dist/guide.md

```diff
@@ -0,0 +1,281 @@
+
+```
+[user@host zod]$ cd ~
+[user@host ~]$ mkdir hello-glob
+[user@host ~]$ cd hello-glob
+[user@host hello-glob]$ mkdir img
+[user@host hello-glob]$ wget -P img https://media.urbit.org/docs/userspace/dist/pot.svg
+[user@host hello-glob]$ tree
+.
+└── img
+    └── pot.svg
+
+1 directory, 1 file
+```
+
+We've grabbed an image to use in our "Hello world!" page. The next thing we need to add is an `index.html` file in the root of the folder. The `index.html` file is mandatory; it's what will be loaded when the app's tile is clicked. Let's open our preferred editor and create it:
+
+```
+[user@host hello-glob]$ vim index.html
+```
+
+In the editor, paste in the following HTML and save it:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <style>
+      div {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <title>Hello World</title>
+  <body>
+    <div>
+      <h1>Hello World!</h1>
+      <img src="img/pot.svg" alt="pot" width="219" height="196" />
+    </div>
+  </body>
+</html>
+```
+
+Our `hello-glob` folder should now look like this:
+
+```
+[user@host hello-glob]$ tree
+.
+├── img
+│   └── pot.svg
+└── index.html
+
+1 directory, 2 files
+```
+
+## Upload to glob
+
+We can now create a glob from the directory. To do so, navigate to `http://localhost:8080/docket/upload` in the browser. This will bring up the `%docket` app's [Globulator](/docs/userspace/dist/glob#globulator) tool:
+
+![Globulator](https://media.urbit.org/docs/userspace/dist/globulator.png)
+
+Simply select the `hello` desk from the drop-down, click `Choose file` and select the `hello-glob` folder in the the file browser, then hit `glob!`.
+
+Now if we return to our ship's homescreen, we should see the tile looks as we specified in the docket file:
+
+![Installed Tile](https://media.urbit.org/docs/userspace/dist/local-install-2.png)
+
+And if we click on the tile, it'll load the `index.html` in our glob:
+
+![Hello World!](https://media.urbit.org/docs/userspace/dist/local-install-3.png)
+
+Our app is working!
+
+## Publish
+
+The final step is publishing our desk with the `%treaty` agent so others can install it. To do this, there's a simple command in the dojo:
+
+```
+> :treaty|publish %hello
+>=
+```
+
+## Remote install
+
+Let's spin up another fake ship so we can try install it:
+
+```
+[user@host hello-glob]$ cd ~/piers/fake
+[user@host fake]$ urbit -F bus
+```
+
+At the time of writing, we must manually request the published apps on a given ship from the dojo. This will likely change in the future but for now we must run the following on `~bus` before we can search for apps on `~zod` from the homescreen:
+
+```
+> :treaty|ally ~zod
+>=
+```
```

Comment:
Note that you can skip this and just `|install ~zod %hello` from the command line, regardless of treaty/ally status.